cluster/router/tag/router.go

```diff
@@ -58,7 +58,7 @@ func (p *PriorityRouter) Route(invokers []protocol.Invoker, url *common.URL, inv
 		return staticTag(invokers, url, invocation)
 	}
 	routerCfg := value.(config.RouterConfig)
```

Comment:
See https://github.com/creasty/defaults/issues/30#issuecomment-942435138. This is a feature for the defaults library.

Reply:
See https://github.com/creasty/defaults/issues/30#issuecomment-942435138. The fixes are for the defaults library. The author suggested to use `*bool` to avoid issues that `false` values can't be set.